reconcile/database_access_manager.py

```diff
@@ -77,6 +77,9 @@ def _get_db(self) -> str:
     def _get_user(self) -> str:
         return self.db_access.username
 
+    def _get_admin_user(self) -> str:
+        return self.admin_connection_parameter.user
```

Comment:
readonly property is more common:

```python
    @property
    def admin_user(self) -> str:
        return self.admin_connection_parameter.user
```

Reply:
Hm, these are not properties of `PSQLScriptGenerator`, but helpers to make accessing related data easier.

Reply:
It's rare in python to define getter functions, https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters.

Such internal helpers can be like this

```python
@property
def _admin_user(self) -> str:
    return self.admin_connection_parameter.user
```